translations.json

```diff
@@ -165,22 +167,25 @@
         },
         "label3.Text": {
             "en": "Status:",
-            "fr-FR": "Etat",
+            "fr-FR": "Etat :",
             "ja-JP": "ステータス :",
             "ko-KR": "상태:",
             "zh-CN": "状态："
         },
         "label4.Text": {
             "en": "Overlay:",
+            "fr-FR": "Overlay :',
             "zh-CN": "悬浮窗："
         },
         "label5.Text": {
             "en": "URL:",
+            "fr-FR": "URL :',
```

Comment:
Is the space here (and in the previous translation) intentional?

Reply:
yeah in french there's always a space before if the symbol have 2 parts like in : ! ; ?